apps/ledger-live-mobile/src/families/avalanchepchain/accountActions.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import type { Account, AccountLike } from "@ledgerhq/types-live";
+import { canDelegate } from "@ledgerhq/live-common/families/avalanchepchain/utils";
+import { Icons } from "@ledgerhq/native-ui";
+import { Trans } from "react-i18next";
+import { AvalanchePChainAccount } from "@ledgerhq/live-common/families/avalanchepchain/types";
+import { NavigatorName, ScreenName } from "../../const";
+
+const getExtraSendActionParams = ({ account }: { account: AccountLike }) => {
+  return {
+    disabled: true,
+    modalOnDisabledClick: null
+  }
+};
+
+const getExtraReceiveActionParams = ({ account }: { account: AccountLike }) => {
+  return {
+    disabled: true
+  }
+};
+
+const getActions = ({ account }: { account: Account }) => {
+  const isDelegationDisabled = !canDelegate(account);
+
+  return [
+    {
+      disabled: isDelegationDisabled,
+      navigationParams: [
+        NavigatorName.AvalancheDelegationFlow,
+        {
+          screen:
+            (account as AvalanchePChainAccount).avalanchePChainResources &&
```

Comment:
I have a little doubt on syntax, do we need to check of avalanchePChainResources exist if we use the ?. operator ?
It's work anyway just so we can keep it but it's just I'm wondering if we couldn't simply write 
`(account as AvalanchePChainAccount).avalanchePChainResources
              ?.delegations.length`

Reply:
I'll update this!